client/my-sites/sidebar/sidebar.jsx

```diff
@@ -5,9 +5,10 @@ var analytics = require( 'lib/analytics' ),
 	classNames = require( 'classnames' ),
 	debug = require( 'debug' )( 'calypso:my-sites:sidebar' ),
 	has = require( 'lodash/has' ),
-	includes = require( 'lodash/includes' ),
-	React = require( 'react' );
+	includes = require( 'lodash/includes' );
```

Comment:
This one is pretty well targeted for eslint fixes, so I'd apply the changes here, then rebase in your in-progress PR after changes here are merged.